pageserver/src/page_service.rs

```diff
@@ -468,8 +473,9 @@ impl postgres_backend::Handler for PageServerHandler {
         } else if query_string.starts_with("basebackup ") {
             let (_, params_raw) = query_string.split_at("basebackup ".len());
             let params = params_raw.split(" ").collect::<Vec<_>>();
+			info!("params.len()={},  params[2].len()={}", params.len(), params[2].len());
             ensure!(
-                params.len() == 2,
+                params.len() >= 2,
```

Comment:
What's going on here? Does this add a new parameter to `basebackup` command?

Reply:
It was just a bug: incorrect  handling of optional third (lsn) parameter of basebackup.
Please notice check `params.len() == 3` few lines later...

Reply:
It was intentional because there was no usages for this third parameter and no tests for that, so I've guarded this case, but didn't remove the following `params.len() == 3`, and marked it with a `TODO`. If I understand it correctly now tests use this option, so I think you can remove the 'TODO'